llpc/tool/amdllpc.cpp

```diff
@@ -1512,23 +1513,52 @@ void findAllMatchFiles(const std::string &inFile,           // [in] Input file n
   }
 
   // Search first file
-  HANDLE pSearchHandle = FindFirstFileA(inFile.c_str(), &data);
-  if (pSearchHandle == INVALID_HANDLE_VALUE) {
+  HANDLE searchHandle = FindFirstFileA(inFile.c_str(), &data);
+  if (searchHandle == INVALID_HANDLE_VALUE) {
     return;
   }
 
   // Copy first file's name
-  pOutFiles->push_back(folderName + data.cFileName);
+  outFiles->push_back(folderName + data.cFileName);
 
   // Copy other file names
-  while (FindNextFileA(pSearchHandle, &data)) {
-    pOutFiles->push_back(folderName + data.cFileName);
+  while (FindNextFileA(searchHandle, &data)) {
+    outFiles->push_back(folderName + data.cFileName);
   }
 
   FindClose(pSearchHandle);
-  return;
 }
 #endif
+
+/// ====================================================================================================================
+/// Expands all input files in a platform-specific way.
+///
+/// @param expanded [out] : Returned expanded input filenames.
+/// @returns Result::Success on success, Result::ErrorInvalidValue when expansion fails.
+static Result expandInputFilenames(std::vector<std::string> &expanded) {
```

Comment:
expanded -> expandedFilenames
I think it is more clear. Change the parameter name in corresponding comments as well.